docs/roadmap.md

```diff
@@ -68,7 +65,7 @@ Those feature are still discussed or designed, not even partly coded.
 
 ## Backlog
 
-This is a kind of whish list, without any priorities, where we try to put some ideas.
+This is a kind of wish list, without any priorities, where we try to put some ideas.
```

Comment:
Should we put this in italic too, like the rest of the content after the title?

Reply:
Yep, we should. I missed this one. Thanks!